projects/commudle-admin/src/app/components/cookie-consent/cookie-consent.component.html

```diff
@@ -0,0 +1,2 @@
+<p>We use cookies to improve your experience using this site.</p>
```

Comment:
Please take a look at the different types of consents here: https://www.cookieconsent.com/

We're using analytics so we'll use the Tracking and Performance one!

Reply:
I've update the text to match the text from cookieconsent.com.